### PR TITLE
Add Kactus.app v0.1.14

### PR DESCRIPTION
**Pull request by imasaru**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t][version-checksum],  
      provide public confirmation by the developer: {{link}}

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

I wasn't sure how to implement the checkpoint, so I'll leave it to @commitay or someone else :)

### CODE REVIEW

File: Casks/kactus.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'kactus' do
+  version '0.1.14'
+  sha256 'cf5ba67e11e9a3ac7583dd5b90d1653d77149a4ec799aebaca3bdc1def12309f'
+
+  # github.com/kactus-io/kactus was verified as official when first introduced to the cask
+  url "https://github.com/kactus-io/kactus/releases/download/v#{version}/Kactus-macos.zip"
+  name 'Kactus'
```
Comment (vitorgalvao) [inline Casks/kactus.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by imasaru** [comment]
I'm guessing you mean that it doesn't need Sketch to run on its own, which, of course, is true, but for the app to work at full functionality, Sketch is a requirement.

Looking at the application:
https://farm5.staticflickr.com/4303/36030060331_9d17097f17_o.png

File: Casks/kactus.rb (hunk 1 above)
Comment (imasaru) [inline Casks/kactus.rb]: Done

**Comment by commitay** [comment]
>  but for the app to work at full functionality, Sketch is a requirement.

Ping @vitorgalvao

**Comment by vitorgalvao** [comment]
@imasaru Ah, alright. Especially since they mentioned planned support for other apps, I though they only needed the files, not the app itself.

**Review by vitorgalvao — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**